QA.md

```diff
@@ -1,31 +1,43 @@
+    1. Now we send the same request but with the correct address that has the available utxos. `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"inputs": [{"type": "query", "filter_address": <address_with_available_utxo>}], "outputs": [{"address": <your_address>, "value": 5}]}' http://localhost:8000/wallet/send-tx`. It must return success true with the transaction detail.
+    1. Send transaction selecting input that was already spent. Get the last sent transaction with `curl -X GET -H "X-Wallet-Id: 123" http://localhost:8000/wallet/tx-history?limit=1` and get the tx hash and index used in this transaction. Use this value to try to send a tx with `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"inputs": [{"hash": <tx_hash>, "index": <input_index>}], "outputs": [{"address": <your_address>, "value": 5}]}' http://localhost:8000/wallet/send-tx`. It must return an error that the input is already spent.
+    1. Send transaction selecting valid input now (you can get a valid utxo with the utxo-filter API) and should return success true.
+    1. Now we will create a new token with `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"name": "Test Token", "symbol": "TST", "amount": 100, "address": <first_address>}' http://localhost:8000/wallet/create-token`. It must return success true and check that the utxo is available with `curl -X GET -H "X-Wallet-Id: 123" http://localhost:8000/wallet/utxo-filter?token=<TOKEN_UID>` and validate the utxo is from the address used in the request. The token_uid is the hash of the transaction created in the first request.
+    1. Now we will mint more tokens with `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"token": <TOKEN_UID>, "amount": 100}' http://localhost:8000/wallet/mint-tokens`. Now check the balance with `curl -X GET -H "X-Wallet-Id: 123" http://localhost:8000/wallet/balance?token=<TOKEN_UID>` the balance must be 200.
+    1. Now we will melt tokens with `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"token": <TOKEN_UID>, "amount": 100}' http://localhost:8000/wallet/melt-tokens`. Now check the balance with `curl -X GET -H "X-Wallet-Id: 123" http://localhost:8000/wallet/balance?token=<TOKEN_UID>` the balance must be 100.
+    1. Now we will create an NFT. `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"name": "Test Token", "symbol": "TST", "amount": 1, "data": "nft data", "address": <first_address>}' http://localhost:8000/wallet/create-nft`. The response must be success true and the created transactions data.
```

Comment:
Should we add this `/create-nft` api in the README.md?

Reply:
Created an issue for that https://github.com/HathorNetwork/hathor-wallet-headless/issues/125